packages/marketplace/src/components/pages/settings/developer-settings-billing-tab/accounts-information-form/direct-debit-section.tsx

```diff
@@ -1,51 +1,41 @@
 import * as React from 'react'
-import { GridItem, FormHeading, FormSubHeading, RadioSelect, Button } from '@reapit/elements'
+import { GridItem, FormHeading, FormSubHeading, Button } from '@reapit/elements'
 import { AccountsInformationFormValues } from './accounts-information-form'
-import formFields from './form-schema/form-fields'
 
 export type DirectDebitSectionProps = {
   setFieldValue: (field: string, value: any, shouldValidate?: boolean | undefined) => void
   values: AccountsInformationFormValues
 }
 
-const hasDirectDebitRadioOptions = [
-  { label: 'Yes', value: 'yes' },
-  { label: 'No', value: 'no' },
-  { label: 'Other (Pay by cheque/bacs/credit)', value: 'other' },
-]
+const DirectDebitSection: React.FC<DirectDebitSectionProps> = ({ values }) => {
+  const { hasReapitAccountsRef, reapitAccountsRef } = values
 
-const { hasDirectDebitField } = formFields
-
-const DirectDebitSection: React.FC<DirectDebitSectionProps> = ({ setFieldValue, values }) => {
-  const { hasDirectDebit } = values
-  const isDirectDebitSectionVisible = hasDirectDebit === 'no'
+  if (hasReapitAccountsRef === 'yes' && reapitAccountsRef.length >= 6)
```

Comment:
Should create a variable for this to make it more readable.
Just like what you've done in `accounts-information-form.tsx`
![image](https://user-images.githubusercontent.com/22409039/85110426-1ac3a500-b23d-11ea-8478-a56c8ffc9e16.png)